### DISCUSSION TITLE
Postcss plugin hides genuine css issues

### DISCUSSION THREAD

**Discussion by pratyushtewari** [microsoft/vscode-discussions#833] [Extension Development QnA] [answered]
Hi,
I want to enable the Postcss plugin but it hides all the errors on my CSS file including the ones I should fix.

This is the screenshot when the plugin is disabled:
![Screenshot 2023-09-21 000042](https://github.com/microsoft/vscode-discussions/assets/251768/10a2cc3d-1b72-49df-806b-b50cc5feb176)

Here is the screenshot after the plugin is enabled:
![Screenshot 2023-09-21 000111](https://github.com/microsoft/vscode-discussions/assets/251768/188a12df-d9ca-4164-aa01-dbabb684e048)

As you can see the `mistake : yes;` css entry is not flagged as an error.

How can I enable postcss plugin but still be able to see these errors?

Thanks,
Pratyush

**Comment by gjsjohnmurray** [comment, marked as answer]
I think you need raise this with the extension author. The forums here are intended for people who are developing VS Code extensions.

**Comment by olguzzar** [comment]
Closing this as off-topic since this forum is specifically for extension development.
